packages/sdk-react/src/components/approvalRequests/consts.ts

```diff
@@ -0,0 +1,36 @@
+import CancelIcon from '@mui/icons-material/Cancel';
+import DangerousOutlinedIcon from '@mui/icons-material/DangerousOutlined';
+import HourglassEmptyIcon from '@mui/icons-material/HourglassEmpty';
+import PaidOutlinedIcon from '@mui/icons-material/PaidOutlined';
+import { ChipTypeMap, type SvgIcon } from '@mui/material';
+
+export enum ApprovalRequestStatus {
+  APPROVED = 'approved',
+  CANCELED = 'canceled',
+  REJECTED = 'rejected',
+  WAITING = 'waiting',
+}
```

Comment:
I would prefer not to use enum at all. Enums cannot be compared to a `string`, they are not tree-shakable, require to be imported in any file you are using them, and add a lot of noise.

```ts
type ApprovalRequestStatus = components['schemas']['ApprovalRequestStatus'];

export const ROW_TO_STATUS_MUI_MAP: {
  [key in ApprovalRequestStatus]: ChipTypeMap['props']['color'];
} = {
  approved: 'success',
  canceled: 'secondary',
  rejected: 'error',
  waiting: 'warning',
};

export const APPROVAL_REQUEST_STATUS_TO_MUI_ICON_MAP: Record<
  ApprovalRequestStatus,
  typeof SvgIcon
> = {
  approved: PaidOutlinedIcon,
  canceled: CancelIcon,
  rejected: DangerousOutlinedIcon,
  waiting: HourglassEmptyIcon,
};
```

In terms of types nothing changed, but less cognitive load.

Reply:
enum removed